examples/src/main/scala/com/basho/riak/spark/examples/dataframes/SimpleScalaRiakDataframesExample.scala

```diff
@@ -0,0 +1,99 @@
+/**
+ * Copyright (c) 2015 Basho Technologies, Inc.
+ *
+ * This file is provided to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License.  You may obtain
+ * a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package com.basho.riak.spark.examples.dataframes
+
+import com.basho.riak.client.core.query.indexes.LongIntIndex
+import com.basho.riak.client.core.query.Namespace
+import com.basho.riak.spark._
+import com.basho.riak.spark.util.RiakObjectConversionUtil
+import org.apache.spark.SparkConf
+import org.apache.spark.SparkContext
+import scala.reflect.runtime.universe
+import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.util.{ Failure, Success }
+import com.basho.riak.client.core.query.RiakObject
+import com.basho.riak.client.api.RiakClient
+import com.basho.riak.client.core.query.Location
+import com.basho.riak.spark.rdd.RiakFunctions
+
+/**
+ * Example shows how Spark DataFrames can be used with Riak
+ * connector for spark
+ */
+object SimpleScalaRiakDataframesExample {
+  private val namespace = new Namespace("users")
+
+  case class UserData(user_id: String, name: String, age: Int, category: String)
+
+  def main(args: Array[String]) {
+    val sparkConf = new SparkConf().setAppName("Riak Spark Dataframes Example")
+
+    setSparkOpt(sparkConf, "spark.master", "local")
+    setSparkOpt(sparkConf, "spark.riak.connection.host", "127.0.0.1:8087")
+
+    val sc = new SparkContext(sparkConf)
+
+    // Work with clear bucket
+    clearBucket(sparkConf)
```

Comment:
I agree, but I wanted to demo how users can save DataFrames into Riak. That's why I create DF on json file and use saveToRiak().